pkgs/development/tools/misc/bashdb/default.nix

```diff
@@ -1,10 +1,15 @@
 { stdenv, fetchurl }:
 
+let
+  basename = "bashdb-4.4-0.92";
```

Comment:
Really! I did not know that. So, by adding the `rec` keyword, attributes can refer to other attributes within the same set? Are there any drawbacks to using the `rec` keyword, or is it to be preferred over the `let` version that I first wrote?